common-rendering/src/components/keyEvents.tsx

```diff
@@ -112,7 +90,50 @@ const listItemStyles = (supportsDarkMode: boolean): SerializedStyles => css`
 
 const timeTextWrapperStyles: SerializedStyles = css`
 	margin-left: ${remSpace[4]};
-	margin-left: 0.5rem;
+`;
+
+const linkStyles = (
+	supportsDarkMode: boolean
+): SerializedStyles => css`
+	text-decoration: none;
+
+	&:hover:before {
+		background-color: ${neutral[46]};
+	}
+
+	&:before {
+		border-radius: 62.5rem;
+		display: block;
+		position: absolute;
+		content: ' ';
+		top: -${remSpace[1]};
+		left: -0.46875rem;
+		width: 0.9375rem;
+		height: 0.9375rem;
+		background-color: ${neutral[86]};
+	}
+
+	${darkModeCss(supportsDarkMode)`
+		&:hover:before {
+			background-color: ${neutral[86]};
+		}
+
+		&:before {
+			background-color: ${neutral[46]};
+		}
+	`}
```

Comment:
So in light mode it's a light grey and then gets darker on hover, and in dark mode it's a dark grey and gets lighter on hover?